docs/user-guide/deploying-application/rollback-deployment.md

```diff
@@ -0,0 +1,34 @@
+# Rollback Deployment
+
+
+Deployments can be rolled back manually. After a deployment is completed, you can manually rollback a stage to a previously deployed image by retaining the same configuration or changing the configuration.
+
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/rollback-deployment/rollback-icon.jpg)
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/rollback-deployment/select-config-to-deploy.png)
+
+1. Select `Rollback` in your deployed pipeline.
+
+2. On the `Rollback` page, select a configuration to deploy from the list:
```

Comment:
Can we take example for example

V1 was deployed then V2 then V3 and then V4

Now if we want to roll back to V2 from V4 then.....

Reply:
We can add an example.

Reply:
Added example.